pkg/kbuffer/revision.go

```diff
@@ -43,10 +43,10 @@ type revisionActivator struct {
 	reporter    StatsReporter
 }
 
-// NewRevisionActivator creates an Activator that changes revision
+// NewRevisionActivator creates an KBuffer that changes revision
 // serving status to active if necessary, then returns the endpoint
 // once the revision is ready to serve traffic.
-func NewRevisionActivator(kubeClient kubernetes.Interface, servingClient clientset.Interface, logger *zap.SugaredLogger, reporter StatsReporter) Activator {
+func NewRevisionActivator(kubeClient kubernetes.Interface, servingClient clientset.Interface, logger *zap.SugaredLogger, reporter StatsReporter) KBuffer {
```

Comment:
Hmm, naming this thing KBuffer feels weird imo. We either need to name it `NewRevisionKBuffer` or keep the whole thing being an "Activator" (since it's role is to wait for the route to be active).